CONTRIBUTING.md

```diff
@@ -26,15 +26,19 @@ brew tap jasonmp85/iwyu
 brew install iwyu
 ```
 
-On Ubuntu you can install it via `sudo apt-get install iwyu`.
+On Ubuntu you can install it via `apt-get`:
+
+```
+sudo apt-get install iwyu
+```
 
 ## Lint Free Code
 
 Automated analysis tools like cppcheck and oclint can point out potential bugs or code that is extremely hard to understand. They also help ensure the code has a consistent style and that it avoids patterns that tend to confuse people.
 
 Ultimately we want lint free code. However, at the moment a lot of cleanup is required to reach that goal. For now simply try to avoid introducing new lint.
 
-To make linting the code easy there are two make targets: `lint` and `lint-all`. The latter does just what the name implies. The former will lint any modified but not committed `*.cpp` files. If there is no uncommitted work it will lint the files in the most recent commit.
+To make linting the code easy there are two make targets: `lint` and `lint-all`. The latter does only what the name implies. The former will lint any modified but not committed `*.cpp` files. If there is no uncommitted work it will lint the files in the most recent commit.
```

Comment:
actually I think `just` is right here, although `exactly` might be better.

Reply:
I agree with that! Made the change.